iconservice/prep/data/prep_container.py

```diff
@@ -53,12 +54,12 @@ def _set_dirty(self, on: bool):
     def is_flag_on(self, flags: 'PRepFlag') -> bool:
         return (self._flags & flags) == flags
 
-    def size(self, active_prep_only: bool = False) -> int:
+    def size(self, active_suspended_prep_only: bool = False) -> int:
```

Comment:
I think 'active_prep_only' is a proper name. In prep container, 'active' refers both ACTIVE and SUSPENDED.

Reply:
I understood why you have defined like this.